### PR TITLE
Add shake-shake loss and accompanying flaky test

### PR DESCRIPTION
**Pull request by dhpitt**
In line with adding more PINN tricks, this is a quick PR that adds shake-shake loss regularization to `neuralop.losses.meta_losses` as `ShakeShakeLoss`.

### CODE REVIEW

File: neuralop/losses/meta_losses.py [hunk 1]
```diff
@@ -87,3 +88,78 @@ def __str__(self):
...
+            from the probability simplex"""
+        weighted_loss = 0.
+        shake_shake_weights = self.sample_from_simplex(len(self.losses))
+
+        for i, loss in enumerate(self.losses):
+            weighted_loss += shake_shake_weights[i] * loss(*args, **kwargs) * self.weights[i]
+        
+        return weighted_loss
+    
+    def update_weights(self, epoch, threshold=0.001):
```
Comment (JeanKossaifi) [inline neuralop/losses/meta_losses.py:139]: This is no longer ShakeShake though.

**Comment by dhpitt** [comment]
Closing for now until a later update.
